JS/imdb.js

```diff
@@ -0,0 +1,143 @@
+
+
+
+//application "state" array 
+var liArray = [];
+
+
+function handle(e){
+  
+  //tirgers the "enter" event
+  if(e.keyCode === 13){
+  	
+  	// Ensure it is only this code that run
+    e.preventDefault(); 
+
+    var catchSearch = document.getElementById("search");
+    
+    // temporary - checking myself
+    console.log(catchSearch.value);
+    console.log(liArray[0].innerHTML.indexOf("e"));
+    
+    //call showList with the text in the searchbox as a value
+    showList(catchSearch.value);
+   }
+  
+ };
+
+function addMovie (e) {
+
+  //assign to movie@ the relevant text element
+  var movieName = document.getElementById("movieName");
+  var movieJanner = document.getElementById("movieJanner");
+  var movieTime = document.getElementById("movieTime");
+  
+  //create a button element that will be used to delete list item
+  var delButton = document.createElement("button");
+  delButton.setAttribute('value', 'X');
+  //delButton.setAttribute('onclick',deleteLi());
+  var buttonData = document.createTextNode("X");
+  delButton.appendChild(buttonData);
+
+
+  //create new "li" item in order to insert it to the existing "ul"
+  var newItem = document.createElement("li");
+
+	 
+  //"create" text node to assign in the new "li" 
+  var newName = document.createTextNode(movieName.value + ", ");
```

Comment:
@oregilron try to create a single text node with all the values and then append it instead of creating a text node for each value, it's unnecessary.